src/relayer/RelayerConfig.ts

```diff
@@ -185,10 +197,15 @@ export class RelayerConfig extends CommonConfig {
         tokenConfig.targetPct = toBNWei(targetPct).div(100);
         tokenConfig.thresholdPct = toBNWei(thresholdPct).div(100);
 
+        if (isDefined(withdrawExcessPeriod)) {
+          tokenConfig.withdrawExcessPeriod = withdrawExcessPeriod;
+        }
```

Comment:
Should be OK to just assign this directly.

```suggestion
        tokenConfig.withdrawExcessPeriod = withdrawExcessPeriod;
```